php/post-types/class-block-post.php

```diff
@@ -41,6 +41,7 @@ class Block_Post extends Component_Abstract {
 	public $pro_controls = array(
 		'repeater',
 		'post',
+		'classic_text',
```

Comment:
Can we please switch these two lines so that Rich Text appears before Classic?

Reply:
Sure

Reply:
It's moved in https://github.com/getblocklab/block-lab/pull/432/commits/1e6aac76933c3a661c69c73d3bb9c9664b81c269